content/preferences-settings/shortcuts.md

```diff
@@ -46,21 +46,35 @@ If you have an external controller, you can trigger discrete actions by pressing
 
 # anatomy of a shortcut
 
-Shortcuts can consist of up to three key/button presses **of the same key/button** (in a quick sequence).
-So, for example, `E`, `E + E` and `E + E + E` are all valid shortcuts to trigger discrete actions, as well as `E + left-click`, `E + right-click` or `E + E + left-click`.
+The simplest shortcut consists of just one key press (e.g., `E`).
+The same key can be repeated up to three times, so also `E + E` and `E + E + E` are valid, and distinct, shortcuts.
+You can further extend a shortcut with up to three clicks of the same mouse button. So, `E + E + left-click` or `E + right-click + right-click` are also valid shortcuts.
+Finally, if your fingers are very nimble and you really need to go there, you can add either or both of the mouse buttons that you haven't incorporated yet. So, also `E + left-click + middle-click` and `E + E + right-click + right-click + right-click + left-click + middle-click` are valid shortcuts.
+Hence, the most complex shortcut can consist of 8 key/button presses:
+* Three key presses of the same `key`, followed by
+* Three presses of any mouse button, followed by
+* 1 press of each remaining mouse button.
 
-On the contrary, `E + A` is not a valid shortcut, and darktable will interpret it as a two different shortcuts: `E` followed by `A`.
+Your shortcut can include one or more modifiers (`Shift`, `Ctrl` and `Alt`). In this case, the modifier(s) have to be held down while executing the remainder of the shortcut. So, `Ctrl + E + E` means holding down `Ctrl` while pressing `E` twice in a rapid sequence.
```

Comment:
I'm not suggesting any wording here; just trying to help improve _your_ mental modal and then you can decide if it is worth explaining to the wider public.

You _don't_ have to press the modifiers before pressing the shortcut keys. You can add them later. So for example if triple-press A is assigned to a slider (and lets assume you have fallbacks enabled). You can do A+A+A hold, then scroll for "normal sized" adjustments. Then (while still holding the A) add Ctrl for some final fine-tuning. Personally I prefer mouse  moves, so I first scroll and then make some final left-right moves at *.1.

Reply:
Shortcuts don't get triggered when you _press_ a key (or button); they get triggered when you _release_ a key (ok, with the exception of _hold_ shortcuts...). Modifiers don't trigger shortcuts, they only "modify" them. So if you press e, then Ctrl then release e, you should get the Ctrl+e shortcut. If you do it _fast_ enough, otherwise it will be a long press or ignored. 

So it has nothing to do with fallbacks but it is more relevant for what you call continuous shortcuts. But also if you combine keypresses with mouse clicks. Ctrl+e+left click you can add the Ctrl _after_ pressing (and holding) the e (_before_ or even _during_ the click).

Disclaimer not behind keyboard so not testing/verifying right now.